features/step_definitions/portfolio.rb

```diff
@@ -1,9 +1,8 @@
 require 'capybara/cucumber'
-require 'capybara/cuprite'
+require 'selenium/webdriver'
 
 Capybara.run_server = false
-Capybara.default_driver = :cuprite
-Capybara.javascript_driver = :cuprite
+Capybara.default_driver = :selenium_chrome_headless
```

Comment:
Please don't change the development environment. It makes tests on the continuous integration server to fail.